docs/articles/install/3-install-with-helm.md

```diff
@@ -77,7 +95,6 @@ By default, the following services will be exposed. You can also choose to overr
 | REST API         | api.$domain        | global.restApiSubdomain        |
 | gRPC API         | agent.$domain      | global.grpcApiSubdomain        |
 | WebSockets API   | websockets.$domain | global.websocketsApiSubdomain  |
-| Status Pages API | status.$domain     | global.statusPagesApiSubdomain |
 | Storage API      | storage.$domain    | global.storageApiSubdomain     |
```

Comment:
should we also add DEX here?

Reply:
Dex is usually exposes on the API subdomain via path prefix

Reply:
yeah, but what I meant is that users will see it as one more Ingress in the k8s cluster even though it uses API subdomain